src/transfer/transfer.model.ts

```diff
@@ -0,0 +1,23 @@
+import * as mongoose from 'mongoose';
+import { Destination, ITransfer } from "./transfer.interface";
+
+const transferScheme: mongoose.Schema = new mongoose.Schema({
+    fileID: {
+        type: String,
+        required: true,
+    },
+    userID: {
+        type: String,
+        required: true,
+    },
+    createdAt: {
+        type: Date,
+        required: true,
+    },
+    destination: {
```

Comment:
Shouldn't you give a default value for `destination`?

Reply:
done.